cmd/snap-bootstrap/cmd_initramfs_mounts_cvm.go

```diff
@@ -308,6 +309,20 @@ func generateMountsModeRunCVM(mst *initramfsMountsState) error {
 		}
 	}
 
+	if createSysrootMount() {
+		// Create unit for sysroot. We restrict this to Ubuntu 24+ for
+		// the moment, until we backport necessary changes to the
+		// UC20/22 initramfs. Note that a transient unit is not used as
+		// it tries to be restarted after the switch root, and fails.
+		rootfsDir := boot.InitramfsDataDir
+		if err := writeSysrootMountUnit(rootfsDir, ""); err != nil {
+			return fmt.Errorf("cannot write sysroot.mount (what: %s): %v", rootfsDir, err)
```

Comment:
Should we wrap the error?
```suggestion
			return fmt.Errorf("cannot write sysroot.mount (what: %s): %w", rootfsDir, err)
```

Reply:
Done